README.md

```diff
@@ -94,15 +94,45 @@ class ClientService:
 Example standalone client, can be used with or without Eventlet:
 
 ``` python
-from example_pb2 import ExampleReply
+from example_pb2 import ExampleRequest
 from example_pb2_grpc import exampleStub
 
 from nameko_grpc.client import Client
 
 with Client("//127.0.0.1", exampleStub) as client:
-    responses = client.unary_stream(ExampleRequest(value="A"))
+    # unary-unary
+    response = client.unary_unary(ExampleRequest(value="a", multiplier=5))
+    print("response:", response.message) # aaaaa
+
+    # unary-stream
+    responses = client.unary_stream(ExampleRequest(value="a", multiplier=5))
+    response_messages = []
+    for response in responses:
+        response_messages.append(response.message)
+    print("responses: ", " | ".join(response_messages)) # aaaaa | aaaaa
+
+    # stream-unary
+    values = (
+        ExampleRequest(value="a", multiplier=5),
+        ExampleRequest(value="b", multiplier=2),
+        ExampleRequest(value="c", multiplier=8),
+    )
+    iterable = iter(values)
+    response = client.stream_unary(values)
+    print("response:", response.message) # aaaaa,bb,cccccccc
+
+    # stream-stream
+    values = (
+        ExampleRequest(value="a", multiplier=5),
+        ExampleRequest(value="b", multiplier=2),
+        ExampleRequest(value="c", multiplier=8),
+    )
+    iterable = iter(values)
+    responses = client.stream_stream(values)
+    response_messages = []
     for response in responses:
-        print(response.message)
+        response_messages.append(response.message)
+    print("responses: ", " | ".join(response_messages)) # aaaaa | bb | cccccccc
```

Comment:
Thanks for expanding these.

Can you perhaps move them to the block above (the nameko service client)? Otherwise it looks like the standalone client has special abilities, whereas it's really just a different usage.

Also I wonder whether using the `@rpc` decorator in that example is confusing, since it's so close to @grpc. It might be better to use `@http` or something obviously different.

Reply:
> Also I wonder whether using the @rpc decorator in that example is confusing, since it's so close to @grpc. It might be better to use @http or something obviously different.

I would include examples for both, it makes it clearer.